cmd/create/cluster/cmd.go

```diff
@@ -1773,7 +1773,11 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 
 		if len(subnets) != len(initialSubnets) {
-			r.Reporter.Warnf("Some subnets have been excluded because they do not fit into the Machine CIDR range")
+			r.Reporter.Warnf("Some subnets have been excluded because they do not fit into chosen CIDR ranges")
+		}
+		if len(subnets) == 0 {
+			r.Reporter.Warnf("No subnets found in current region that are valid for the chosen CIDR ranges")
+			confirm.Prompt(false, "Continue with default VPC cluster flow?")
```

Comment:
![image](https://github.com/openshift/rosa/assets/14313111/28d2396f-5157-4a7a-960a-d176c039a48b)

Maybe a slight rewording to better match the UI.

```
Continue with default, a new VPC will be created for your cluster.
```
Or something like that